fem/eltrans.cpp

```diff
@@ -200,24 +200,65 @@ void IsoparametricTransformation::Transform (const DenseMatrix &matrix,
    }
 }
 
+const IntegrationPoint IsoparametricTransformation::FindInitialIntegrationPoint(
```

Comment:
Sounds like a good idea.  So, should I make this function ``public`` instead of ``private``?

Also, are you suggesting that this be a new parameter for ``TransformBack()``, or for the ``IsoparametricTransformation`` class?  

The former is a virtual function of the ``ElementTransformation`` class.

Reply:
Yes, I was suggesting to make `FindClosestRefinedPoint()` public and add the refinement parameter to it. 

This could also be a parameter of `TransformBack()` that is being passed down, so one can revert to testing the center only (ref = 0) if they want, e.g. in

https://github.com/mfem/mfem/blob/c88f645e6150fa74228e0c95170804987e620d8b/mesh/mesh.cpp#L8570